### ISSUE TITLE
Unable to download and install from git-hub

### ISSUE THREAD

**Issue by pjhLongview** [zioth/dokuwik-anchor#5]
When I try in download and install from DW, I get the following error message:

Unable to download the file: https://github.com/zioth/dokuwiki-anchor/zipball/master

Am I missing something?
Peter Hunter
Longview Ranch

**Comment by zioth** [comment]
Must have been a temporary glitch in github. I just tried it, and it works.

**Comment by pjhLongview** [comment]
Something must be wrong with by DW installation.  None of the plugins will update or installed.  I did not have a self-signed certificate, which I created and installed on the site, but even accessing via https gave me the same error.  The site in on a Win Server 2016 running IIS 10.
I was able to install your plugin by downloading it directly from GitHub and extracting it to /lib/plugins/anchor.  Problem solved for now, but I'm confused why I can't seem to install it directly from the extension manager.
